types/babel-core/index.d.ts

```diff
@@ -10,10 +10,12 @@ export { t as types };
 export type Node = t.Node;
 export import template = require('babel-template');
 export const version: string;
-import traverse, { Visitor } from "babel-traverse";
+import traverse, { Visitor, NodePath } from "babel-traverse";
```

Comment:
This `NodePath` isn't used. Did you want to use it for `wrapPluginVisitorMethod`?

The rest looks great.

Reply:
Left over from my trying to get `wrapPluginVisitorMethod` to have better specified types – was going to ask about that actually. The callback function it takes and the function it returns can take any `NodePath` variant. I couldn't come up with a way to add concrete typing to those functions without enumerating every templated type, like is done in `babel-traverse`. Do you have a better way to do this?

Reply:
Simple `NodePath` (AKA `NodePath<Node>`) is okay in this case.